atom/browser/api/atom_api_window.cc

```diff
@@ -92,6 +95,16 @@ void Window::WillCloseWindow(bool* prevent_default) {
   *prevent_default = Emit("close");
 }
 
+void Window::OnFrameRendered(scoped_ptr<uint8[]> rgb, const int size) {
+  v8::Locker locker(isolate());
+  v8::HandleScope handle_scope(isolate());
+
+  v8::Local<v8::ArrayBuffer> data = v8::ArrayBuffer::New(isolate(), rgb.get(), size);
```

Comment:
`blink` requires creating `v8::ArrayBuffer` from its own API, otherwise it would crash when you print it in the console of devtools. We can just use `node::Buffer::New` here, which is safe and more easy to use.